test/assert.js

```diff
@@ -74,6 +74,26 @@ assert.holdsItem = function (pokemon, message) {
 	});
 };
 
+assert.trapped = function (fn) {
+	assert.throws(() => fn(), Error, "[Invalid choice] Can't switch: The active Pokémon is trapped");
+};
+
+assert.cantMove = function (fn, pokemon, move) {
+	if (pokemon && move) {
+		assert.throws(() => fn(), Error, new RegExp(`\\[Invalid choice\\] Can't move:.*${pokemon}.*${move}`, 'i'));
+	} else {
+		assert.throws(() => fn(), Error, new RegExp(`\\[Invalid choice\\] Can't move:`));
+	}
+};
+
+assert.cantUndo = function (fn) {
+	assert.throws(() => fn(), Error, new RegExp("\\[Invalid choice\\] Can't undo:"));
```

Comment:
In hindsight I'm confused why this isn't just a regex literal.

```suggestion
	assert.throws(() => fn(), Error, /\[Invalid choice\] Can't undo:/);
```

Reply:
It should be. I did the `cantUndo` case with interpolation that actual required the constructor first, and then just copied and pasted the rest. Fixed this and above.